### PR TITLE
Internal: Infra  QUnit tests elements helper refactor. [ED-6418]

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: tests/qunit/tests/assets/dev/js/editor/document/elements/helper.js [hunk 1]
```diff
@@ -1,4 +1,60 @@
...
+
+	/**
+	 * @return {Container}
+	 */
+	static createAutoButton( eContainer = null, settings = {} ) {
+		if ( ! eContainer ) {
+			eContainer = this.createSection( 1, true );
+		}
+
+		return this.createWidgetButton( eContainer, settings );
```
Comment (ItzhakAvm) [inline tests/qunit/tests/assets/dev/js/editor/document/elements/helper.js:45]:
```suggestion
		return this.createAuto( 'widget' );
```

**Review by ItzhakAvm — CHANGES_REQUESTED** [review]

**Review by YitzhakAvm — APPROVED** [review]

**Comment by iNewLegend** [comment]
@matipojo, Please review

File: tests/qunit/tests/assets/dev/js/editor/document/elements/commands/paste-style.spec.js [hunk 2]
```diff
@@ -39,7 +45,10 @@ export const PasteStyle = () => {
 
 			QUnit.test( 'History', ( assert ) => {
 				const eWidgetSimple = ElementsHelper.createAutoButton(),
-					eWidgetStyled = ElementsHelper.createAutoButtonStyled(),
+					eWidgetStyled = ElementsHelper.createAutoButton( null, {
```
Comment (matipojo) [inline tests/qunit/tests/assets/dev/js/editor/document/elements/commands/paste-style.spec.js:48]:
```suggestion
					eWidgetStyled = ElementsHelper.createWrappedButton( null, {
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/qunit/tests/assets/dev/js/editor/document/elements/commands/paste-style.spec.js (hunk 2 above)
Comment (iNewLegend) [reply tests/qunit/tests/assets/dev/js/editor/document/elements/commands/paste-style.spec.js]: Handled

**Review by matipojo — CHANGES_REQUESTED** [review]
Please resolve the conflicts

**Pull request merged**